exercises/flatten-array/flatten-array_spec.lua

```diff
@@ -33,4 +33,11 @@ describe('flatten-array', function()
       flatten({ { { {  } } }, { { } } })
     )
   end)
+
+  it('should flatten array from readme example', function()
+    assert.are.same(
+      {1,2,3,4,5},
+      flatten({1,{2,3,nil,4},{null},5})
```

Comment:
Unfortunately the README is inherited from x-common: https://github.com/exercism/x-common/blob/master/exercises/flatten-array/description.md.

I'm not opposed to introducing a custom "nil" value here. It would be a good way to introduce that topic